RecoParticleFlow/PFClusterProducer/plugins/RecHitCreators.cc

```diff
@@ -12,8 +13,8 @@
 
 
 EDM_REGISTER_PLUGINFACTORY(PFRecHitFactory, "PFRecHitFactory");
-DEFINE_EDM_PLUGIN(PFRecHitFactory, PFEERecHitCreator, "PFEERecHitCreator");
-DEFINE_EDM_PLUGIN(PFRecHitFactory, PFEBRecHitCreator, "PFEBRecHitCreator");
```

Comment:
PFEBRecHitCreator and PFEERecHitCreator are still used (HLT configuration) and apparently you have not removed the code for them either.
We can go all the way and remove PFEcalRecHitCreator implementation and propagate also customization to HLT.
A simpler alternative is to keep these in now, get the HLT to migrate to the new ones and then remove.